File: wvcore/src/main/java/com/lds/wvrss/ui/ChromeView.java
package com.lds.wvrss.ui;

import android.app.Activity;

public interface ChromeView {

    void removeErrorPage();

    void onPageFinished(String url);

    void openBrowser(Activity context, String url);

    void webClientError(ReceivedError failure);

    void setErrorPage(ReceivedError receivedError);
}

File: webview/src/main/java/com/walhalla/webview/ChromeView.kt
package com.walhalla.webview

import android.app.Activity
import android.content.Context

interface ChromeView {
    fun onPageStarted(url: String?)

    fun onPageFinished( /*WebView view, */url: String?)

    fun webClientError(failure: ReceivedError)

    fun removeErrorPage()

    fun setErrorPage(receivedError: ReceivedError)

    fun openBrowser(url: String)

    //void openOauth2(Activity context, String url);
    //void mAcceptPressed(String url);
    //void eventRequest(BodyClass bodyClass);
}